rpc_jobs/rpc_aio.yml

```diff
@@ -225,8 +226,10 @@
             }} catch (e) {{
               print(e)
               throw e
+              currentBuild.result = 'FAILURE'
```

Comment:
Oh, I see what you mean, you are right, currentBuild.result should be set before throw here.